src/controllers/tweets.controller.js

```diff
@@ -0,0 +1,133 @@
+const tweetsService = require('../services/tweets.service');
+const authentication = require('../middlewares/authentication.middleware');
+
+// const invalidIdMsg = 'argument passed in must be a string of 12 bytes or a string of 24 hex characters or an integer';
+const invalidIdMsg = 'آرگومان وارد شده باید یک رشته از 12 بایت یا یک رشته از 24 کاراکتر هگزا داشته باشد یا یک عدد باشد';
+// const someThingWentWrongMsg = 'something went wrong';
+const someThingWentWrongMsg = 'مشکلی پیش آمده است، لطفا دوباره امتحان کنید';
+// const tweetNotFoundMsg = 'tweet not found';
+const tweetNotFoundMsg = 'توییت یافت نشد';
+
+module.exports = {
+    getAllTweets(req, res) {
+        tweetsService.getAllTweetsService()
+            .then(tweets => {
+                res.send(tweets);
+            })
+            .catch(err => {
+                console.log(err);
+                res.status(422).json({
+                    message: someThingWentWrongMsg
+                });
+            });
+    },
+
+    getTweetById(req, res) {
+        const id = req.params.id;
+
+        tweetsService.getTweetByIdService(id)
+            .then(tweet => {
+                if (tweet) {
+                    res.send(tweet);
+                } else {
+                    res.status(404).json({
+                        message: tweetNotFoundMsg
+                    });
+                }
+            })
+            .catch(err => {
+                console.log(err);
+                res.status(400).json({
+                    message: invalidIdMsg
+                });
+            });
+    },
+
+    createTweet(req, res) {
+        const tweetProps = req.body;
+
+        tweetsService.createTweetService(tweetProps)
+            .then(tweet => {
+                if (tweet) {
+                    res.send(tweet);
+                } else {
+                    res.status(422).json({
+                        message: someThingWentWrongMsg
+                    });
+                }
+            })
+            .catch(err => {
+                console.log(err);
+                res.status(422).json({
+                    message: someThingWentWrongMsg
+                });
+            });
+    },
+
+    deleteTweet(req, res) {
+        const id = req.params.id;
+
```

Comment:
I think we should check that the creator of the tweet is the same user who logged in.